server/src/printers/TestsPrinter.cpp

```diff
@@ -167,7 +167,11 @@ void TestsPrinter::printLazyVariables(const Tests::MethodDescription &methodDesc
         const auto &param = testCase.lazyVariables[i];
         const auto &value = testCase.lazyValues[i];
         if (param.type.isObjectPointer()) {
-                strDeclareVar(param.type.baseType(), param.varName, value.view->getEntryValue());
+                if (!types::TypesHandler::baseTypeIsVoid(param.type)) {
+                    strDeclareVar(param.type.baseType(), param.varName, value.view->getEntryValue());
+                } else {
+                    strDeclareVar("unsigned char", param.varName, value.view->getEntryValue());
```

Comment:
Since we do not know for sure what is the exact pointed type and its size, such tests can lead to false errors during the launch. 
Would be better to skip them as done in #200?

Reply:
You might also want to add a new kind of type [here](https://github.com/UnitTestBot/UTBotCpp/blob/3cdb310e1802067d7705c2524f9163880b7850bb/server/src/types/TypeVisitor.cpp#L33) for the ease of distinguishing incomplete types

Reply:
I think, that void* point on any data, and this data is no less than unsigned char. So we use unsigned char

Reply:
What happens if it points to data which is more than one byte? Does regression test fails due to out of bound access?

Reply:
Following the changes, `void` is just replaced with `char` and never with `char array` in the example from the issue, right? How do you handle the case when it points to larger objects?

Reply:
We replace void with char, because in ktestjson file unnamed void has size equals to 1 byte.

Reply:
So if we use more than 1 byte we give error

Reply:
It could be not an error in symbolic execution, so it leads to a false error in concrete execution and confuses the user

Reply:
I don't know, how klee works with void*, but maybe problem in klee. If we use more than 1 byte, then klee doesn't know about that?